rfcs/spinner-component.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Spinner} from 'baseui/spinner';
+
+export default () => {
+  return
+    <Spinner
+      variant={Spinner.VARIANT.INDETERMINATE}
+      size={10}
+    />;
+};
+```
+
+## Exports
+
+* `Spinner`
+* `VARIANT`
+
+## `Spinner` API
```

Comment:
Would be great to have a `text` prop also which is placed below the spinner for loading text.

In addition, a `success` variant would also be useful (see our usage below).

We have built a custom spinner component internally in BaseUI theme in pure CSS, but we don't have the bandwidth to contribute back for a couple of weeks. Do let us know if you need help or want to see our implementation. Cheers!

![spinner-with-success mov](https://user-images.githubusercontent.com/360201/45134913-75038b00-b1ba-11e8-99ea-82801763a797.gif)

Reply:
cool, thanks David - I'll update the PR, and ping you
also, will follow up in private re your implementation

Reply:
should the success variant be a separate component, maybe an icon?